lua/dap-vscode-js/init.lua

```diff
@@ -12,7 +12,7 @@ local js_dap = require("dap-vscode-js.dap")
 local dapjs = {}
 
 ---Setup adapter and/or configs
----@param settings string
+---@param settings table
```

Comment:
It seems like there's actually a `@class` definition for this:

```suggestion
---@param settings Settings
---@param force boolean?
```

Reply:
you're right, thanks :)